sdk/python/kfp/components/executor.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 import json
 import inspect
-from typing import Any, Callable, Dict, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
```

Comment:
Is `List` imported in `io_types` ? Otherwise it doesn't get included in the component implementation.

Reply:
Yes, it is. (Noticed that the imports here are not included with the `import re`).